shoop_tests/core/test_order_creator.py

```diff
@@ -57,7 +58,10 @@ def test_order_creator(admin_user):
         require_verification=True,
     ))
 
-    creator = OrderCreator(request=None)
+    request = RequestFactory().get("/")
```

Comment:
```python
request = RequestFactory().get("/")
request.shop = get_default_shop()
```

is copypasted into ALL THE PLACES. Any way we could refactor it? (FWIW, there's `apply_request_middleware`, which will install all the usual attributes into a request...)